.travis.yml

```diff
@@ -8,6 +8,10 @@ os:
   - windows
   - linux
 
+branches:
+  only:
+    - master
```

Comment:
are you sure? would be nice to know if tests pass on PRs

Reply:
PR builds still trigger (e.g. this PR), but branches that do not have PRs won't, and you won't get 2 builds per branch